content/en/blog/_posts/2021-08-20-seccomp-default.md

```diff
@@ -0,0 +1,253 @@
+---
+layout: blog
+title: "Hardening Kubernetes security by enabling seccomp by default"
+date: 2021-08-20
+slug: seccomp-default
+---
+
+**Author:** Sascha Grunert, Red Hat
+
+_Editor's note: If you have any questions or feedback about this blog post, feel
+free to reach out via the [Kubernetes slack in #sig-node][slack]._
+
+[slack]: https://kubernetes.slack.com/messages/sig-node
+
+---
+
+Seccomp is a security mechanism for Linux processes to filter system calls
+(syscalls) based on a set of defined rules. Applying seccomp profiles to
+containerized workloads is one of the key tasks when it comes to enhancing the
+security of the application deployment. Developers, site reliability engineers
+and infrastructure administrators have to work hand in hand to create,
+distribute and maintain the profiles over the applications life-cycle.
```

Comment:
Let's mention that this article is about a change that came in the v1.22 release (that's right isn't it?)

Right now we make the reader go through quite a bit of text before they see that detail.